frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -406,6 +406,10 @@ export const CreateVMWizardPageComponent: React.FC<CreateVMWizardPageComponentPr
           prop: VMWizardProps.commonTemplates,
           matchLabels: { [TEMPLATE_TYPE_LABEL]: TEMPLATE_TYPE_BASE },
         }),
+        getResource(PersistentVolumeClaimModel, {
+          namespace: 'openshift-cnv-base-images',
+          prop: VMWizardProps.openshiftCNVBaseImages,
```

Comment:
this should be in relevant loadingResources of components which affect the updaters

Reply:
added :+1: